etc/bash-utils.sh

```diff
@@ -143,11 +150,18 @@
 
 should-build-rmm() {
     update-environment-variables $@ >/dev/null;
-    $(should-build-cudf) || [ "$BUILD_RMM" == "YES" ] && echo true || echo false;
+    $(should-build-cuml) || $(should-build-cugraph) || $(should-build-cuspatial) || $(should-build-cudf) || [ "$BUILD_RMM" == "YES" ] && echo true || echo false;
```

Comment:
This is implied behavior from `should-build-cudf` already. It's fine to make explicit, just wanted to note that it's not changing behavior.

Reply:
Yup I just wanted to make it explicit.